genienlp/tasks/almond_task.py

```diff
@@ -240,9 +236,7 @@ def pad_features(self, features, max_size, pad_id):
         return features
 
     def oracle_type_ids(self, tokens, entity2type):
```

Comment:
Do you still need the oracle? It's so hacky, I wonder if we can do it with a separate script that generates oracle bootleg_features.jsonl

Reply:
That's a good idea but it needs some additional work. I'll revisit it later if we decide to keep it after submission.